app/components/sidebars/main/channels_list/channel_item/channel_item.test.js

```diff
@@ -43,4 +43,17 @@ describe('ChannelItem', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should match snapshot for deactivated user', () => {
+        const newProps = {
+            ...baseProps,
+            teammateDeletedAt: 100,
+            type: 'D',
+        };
+        const wrapper = shallow(
+            <ChannelItem {...newProps}/>,
+            {context: {intl: {formatMessage: jest.fn()}}},
+        );
+        expect(wrapper).toMatchSnapshot();
```

Comment:
I thought about it when I saw the PR. I didn't make the change as I want to understand the cons to it because I don't see this pattern in other projects around. 
That change I think needs a good evaluation and an addition to developer decision doc if agreed. 
We can wait for the other PR and change this or make this change in the other PR if this gets in first

Reply:
Right, I'll queue that topic.
The snapshots on the webapp is fairly different than here.  Here, I noticed that the big chunk on what we're capturing is react-native specific implementation and it makes it hard to recognize whether it's really that we expected.  I wonder if someone looks at the snapshot and validate after doing so.